Stratis.Bitcoin/RPC/Controllers/FullNodeController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NBitcoin;
+using Stratis.Bitcoin.RPC.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Stratis.Bitcoin.RPC.Controllers
+{
+	//TODO: Need to be extensible, should be FullNodeController
+	public partial class ConsensusController : Controller
+	{
+		[ActionName("stop")]
+		public Task Stop()
+		{
+			_FullNode.Dispose();
+			return Task.CompletedTask;
+		}
+
+		[ActionName("getrawtransaction")]
+		public async Task<TransactionModel> GetRawTransaction(string txid, int verbose = 0)
+		{
+			uint256 trxid;
+			if (!NBitcoin.uint256.TryParse(txid, out trxid))
+				throw new ArgumentException(nameof(txid));
+
+			ChainedBlock block = null;
+			Transaction trx = (await _FullNode.MempoolManager.InfoAsync(trxid))?.Trx;
```

Comment:
One observation is that the mempool and store may not always be present (they might be null) if the feature is not registered in the builder, put null checks around that.

You might also want to check if Setting.Txindex = false and not bother to try to fetch the trx from store as it wont be indexed.